modules/core/test/com/haulmont/cuba/core/MessagesTest.java

```diff
@@ -111,6 +112,25 @@ public void testMultiInclude() {
         assertEquals(msg3, "overridden");
     }
 
+    @Test
+    public void testIncludeDefaultLoc() {
+        Messages messages = AppBeans.get(Messages.class);
+        TestMessageTools messageTools = (TestMessageTools) AppBeans.get(MessageTools.class);
```

Comment:
Replace use of `TestMessageTools` with `UserSessionSource#getUserSession()#getLocale()` - as you did in previous version